helix-view/src/document.rs

```diff
@@ -562,13 +562,26 @@ impl Document {
 
             // TODO Temporary file creation is a workaround to solve large file corruption
             // that occurs when crashing during a file save
-            let mut tmp_path = path.clone();
-            tmp_path.set_file_name(format!(".~{}", path.file_name().unwrap().to_str().unwrap()));
+            let tmp_path = PathBuf::from(std::env::var("HOME").unwrap())
```

Comment:
You can use this function instead: https://github.com/helix-editor/helix/blob/a3ed9169df8d0a56ab4f698d2c90e7e6b4404633/helix-loader/src/lib.rs#L70-L76

Reply:
It's already created on startup as it's the folder where the log is written.

Reply:
oooh, love it. Updated to that and removed the bit where it tests for the directory since it's already there